agent/unit-test/softwarecontaineragent_unittest.cpp

```diff
@@ -36,18 +35,20 @@ class SoftwareContainerAgentTest: public ::testing::Test
     bool m_shutdownContainers = true;
     int m_shutdownTimeout = 2;
     std::shared_ptr<Workspace> workspace;
+    std::string m_configPath =
+        "/home/vagrant/softwarecontainer/agent/unit-test/test-data/CS_unittest_ServiceManifest.json";
```

Comment:
It will be equally bad if someone doesn't build softwarecontainer in vagrant, or in some other path in the VM. We can't assume everyone will build in a VM. You should be able to point to the tests relative to the test binary. It is reasonable to assume that test-data directory is available in the same place as the test binary.